src/components/StyleGuide/Colors.mdx

```diff
@@ -46,6 +52,59 @@ transparency are tough to talk about when it comes to color combinations and
 color contrast. That being the case, we have used question marks for the
 contrast values and WCAG status.
 
+## Primitives
+
+The primitive colors and the related colors scales were reverse engineered using
+the legacy semantic color palette. Because of that, the primitive colors are not
+used extensively throughout the semantic color assignments.
```

Comment:
This answers my question about why they're not "root colors" or used in the base semantic tokens.